python/tank/bootstrap/cached_configuration.py

```diff
@@ -245,37 +249,6 @@ def update_configuration(self):
         # make sure tank command and interpreter files are up to date
         self._config_writer.create_tank_command()
 
-        if self._pipeline_config_id:
```

Comment:
This was leftover code from an intermediary version of the boostrapper that should not have been kept. It tried to update the pipeline configuration field in Shotgun, which it should never have.